setup.py

```diff
@@ -94,7 +94,7 @@ def read(*names, **kwargs):
     )
 
     import torch
-    from torch.utils.cpp_extension import BuildExtension, CUDAExtension
+    from torch.utils.cpp_extension import BuildExtension, CUDAExtension, CUDA_HOME
```

Comment:
Curious as to what this import does.

Reply:
Ah, you're right. I can take that out now. I used it in an earlier version when checking for whether the local machine has CUDA installed and setup properly. It tries to find the CUDA install path: https://pytorch.org/docs/stable/_modules/torch/utils/cpp_extension.html#CppExtension (see _find_cuda_home()